src/pages/hegel/reference/mechanical-object-section-1.mdx

```diff
@@ -0,0 +1,18 @@
 crucial to understanding the logical structure of the “mechanical object”. The first thing to clarify is what Hegel means by ‘its determinateness’. The moment of ‘universality’ is the core of the object’s identity and that is a moment of self-relating negativity. When the determination of “universality” was examined in the beginning of the “Doctrine of the Concept” it was comprehended as being a self-relating negativity . This negativity that remains within the “universal” is the determinateness [*Bestimmtheit*] of the “universal”. Next, we have to clarify what Hegel means by ‘its universality’ because I have already stated that the “mechanical object” is not just a “universal” but is immediately all three determinations of the “Concept”. So what does Hegel mean by ‘its universality’. The ‘universality’ of a determination is its core essence – it is what it is for that determination to be what it is. Just as the universal determination of a chair is that it has ‘the capacity to support someone in a seated position’, for example, so too do abstract determinations also refer to their moment of universality as the source of their identity. Thus, in the case of the “mechanical object”, when Hegel refers to ‘its universality’ he is referring to its basic source of identity. Now that we have clarified what Hegel means by ‘its determinateness’ and ‘its universality’, we are in a good position to understand the rest of that sentence. The determinateness, i.e. the negativity, of the “mechanical object” does not reflect outwards. In other words, it does not relate to anything that is outside of itself. Rather, it is reflected into its “universality”, i.e. it is reflected into its own source of identity. One simplistic way to put this is that mechanical objects are narcissistic – they do not relate to anything outside of themselves and are only self-relating. It is in this sense that the “mechanical object” is both indeterminate and has difference within it. It has difference within it because of its determinateness, because of its negative relation to itself. In this moment of negative self-relation, the “mechanical object” is immediately identical to all three determinations of the “Concept”. It is indeterminate because it does not relate to anything outside of itself, it is only self-relating. In its self-relating it is, effectively, relating to itself as the immediate identity of the determinations of the “Concept”. It is for this reason that Hegel writes the following: “Because this indeterminate determinateness is essential to the object, the latter is within itself a *plurality* of this kind, and must therefore be regarded as a *composite* or *aggregate*” (712).  What it is for the “mechanical object” to be is for it to relate to itself as the immediate identity of other objects (the determinations of the “Concept”). In other words, what it is for it to be itself is for it to be itself through other objects that are taken to be identical to itself. 
```

Comment:
1. Long, heavy paragraph. Can it be chopped up? I think you do a great job working through the material, but it is a tough read still. 
2. “Doctrine of the Concept” => does this need quotation marks?
3. "The ‘universality’ of a determination is its core essence – it is what it is for that determination to be what it is." => loosely speaking, I kind of see what you're getting at, but when looked at these terms technically, `essence` and `what is` (which seems to signify `being`) don't seem to cohere.

Reply:
1. Yep, can definitely break it into 3 paragraphs.
2. Again, I do that as a way of distinguishing between logical use and general use. Refer to point 2 in the above comments box - happy to drop this stylistic feature.
3. I guess that's an interesting point. One thing to say is the following, isn't the Concept the unity of Being and Essence? Not sure, what to make of that in terms of the above, but that's the first thing that came to mind. 
More loosely, I use the "essence" language because Hegel refers to it as an "essential determinateness" and I use the "being" language because this is ontology, so at every step we are thinking about what it is for a particular determination to be. I have this itch that there is a bigger discussion to be had about this, but I can't quite pinpoint it.

Reply:
1 and 2 ✅

3. Ok, we'll leave it for now and can open a discussion later on it. ✅